### ISSUE TITLE
NPE with extern enum abstracts

### ISSUE THREAD

**Issue by kaikoga** [HaxeFoundation/intellij-haxe#971]
```
package ;

import flash.utils.Endian;

class Sample {
	public function new() {
		var a = Endian.BIG_ENDIAN;
	}
}
```

Plugin version: v1.2 for IDEA 2018.x
IDEA version: IDEA Ultimate 2018.3.6
OS: macOS Mojave 10.14.6

Stacktrace:

```
java.lang.NullPointerException
	at com.intellij.plugins.haxe.model.HaxeImportModel.lambda$getExposedMember$5(HaxeImportModel.java:164)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174)
	at java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1353)
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
	at com.intellij.plugins.haxe.model.HaxeImportModel.getExposedMember(HaxeImportModel.java:165)
	at com.intellij.plugins.haxe.model.HaxeImportModel.exposeByName(HaxeImportModel.java:143)
	at com.intellij.plugins.haxe.util.HaxeResolveUtil.lambda$searchInImports$3(HaxeResolveUtil.java:681)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174)
	at java.util.stream.IntPipeline$4$1.accept(IntPipeline.java:250)
	at java.util.stream.Streams$RangeIntSpliterator.tryAdvance(Streams.java:89)
	at java.util.stream.IntPipeline.forEachWithCancel(IntPipeline.java:162)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
	at com.intellij.plugins.haxe.util.HaxeResolveUtil.searchInImports(HaxeResolveUtil.java:684)
	at com.intellij.plugins.haxe.util.HaxeResolveUtil.tryResolveClassByQNameWhenGetQNameFail(HaxeResolveUtil.java:649)
	at com.intellij.plugins.haxe.util.HaxeResolveUtil.tryResolveClassByQName(HaxeResolveUtil.java:569)
	at com.intellij.plugins.haxe.lang.psi.HaxeResolver.checkIsClassName(HaxeResolver.java:202)
	at com.intellij.plugins.haxe.lang.psi.HaxeResolver.doResolve(HaxeResolver.java:115)
	at com.intellij.psi.impl.source.resolve.ResolveCache.lambda$resolve$0(ResolveCache.java:150)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:98)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolve(ResolveCache.java:149)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:239)
	at com.intellij.plugins.haxe.lang.psi.HaxeResolver.resolve(HaxeResolver.java:77)
	at com.intellij.plugins.haxe.lang.psi.impl.HaxeReferenceImpl.multiResolve(HaxeReferenceImpl.java:215)
	at com.intellij.plugins.haxe.lang.psi.impl.HaxeReferenceImpl.resolveToComponentName(HaxeReferenceImpl.java:232)
	at com.intellij.plugins.haxe.ide.annotator.HaxeColorAnnotator.annotate(HaxeColorAnnotator.java:65)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.runAnnotators(DefaultHighlightVisitor.java:120)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:87)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.runVisitors(GeneralHighlightingPass.java:353)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$collectHighlights$5(GeneralHighlightingPass.java:286)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:313)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$analyzeByVisitors$6(GeneralHighlightingPass.java:316)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:71)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:316)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectHighlights(GeneralHighlightingPass.java:283)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:227)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:69)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:423)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1168)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:416)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:415)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:391)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:147)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:222)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:389)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:161)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157
```

**Comment by EricBishton** [comment]
Looks like HaxeImportModel.java:157 is being passed a null string.  Got to figure out why that is.  Protecting from doing the loop when the passed null is the easy part.  Something upstream is probably breaking as well, though with this stack trace, it shows things just won't colorize properly.

**Comment by EricBishton** [comment]
I did a fix in 12bc205 that solves the problem.  You can pick up a test build at https://github.com/EricBishton/intellij-haxe/releases/tag/Post1.2TestBuild.2.  It will be in release 1.3.

**Comment by kaikoga** [comment]
Thank you for response, I have just tried the test build but noticed extern enum abstracts in std lib just failed to parse at all.
I'm no longer having any NPEs btw.

<img width="530" alt="image" src="https://user-images.githubusercontent.com/4142423/71447448-c77f0700-2771-11ea-94bb-9d00a42d3ea8.png">
